### DISCUSSION TITLE
cant record workflow with existing workflow

### DISCUSSION THREAD

**Discussion by fiqryq** [AutomaApp/automa#982] [Q&A] [answered]
how can I record workflow with existing workflow, I can't find the menu because every time I click record on this menu I always get redirected to documentation.

<img width="431" alt="image" src="https://user-images.githubusercontent.com/25787603/205336467-aa439b42-f978-45c2-957b-fcbbafc95da9.png">

**Comment by Kholid060** [comment, marked as answer]
![image](https://res.cloudinary.com/chat-story/image/upload/v1666315751/automa/start-recording_hpfg9u.gif)
